applications/nrf5340_audio/src/main.c

```diff
@@ -207,6 +220,19 @@ void main(void)
 
 	ret = audio_datapath_tone_play(440, 500, 0.2);
 	ERR_CHK(ret);
+	/* Enable MCUMGR */
+	#ifdef CONFIG_MCUMGR_CMD_IMG_MGMT
```

Comment:
Nitpick: Swap the places of CONFIG_MCUMGR_CMD_IMG_MGMT and CONFIG_MCUMGR_CMD_OS_MGMT to match the order of #include on line 29-34

Reply:
Re-order function register sequence.